doc/userguide/rules/tls-keywords.rst

```diff
@@ -136,6 +136,21 @@ Usage::
 
   tls_cert_valid;
 
+tls.cert
+--------
+
+Do a "raw" match on the TLS certificate. This keyword only matches on the
+first certificate in the certificate chain.
```

Comment:
no it doesn't, it matches on all of them... right?

Reply:
Yeah. I should probably fix that!